docs/tutorials/configure/scuttlebot.md

```diff
@@ -119,7 +88,7 @@ Add the following JSON objects to the `components` array:
       "board": "local",
```

Comment:
You can remove this line; the encoder no longer depends on the board. Same comment on line 102, below.